src/utils/helpers/structure/generateReviewProgress.ts

```diff
@@ -73,15 +73,14 @@ const getSums = (elements: PageNEW[]) => {
     doneNoneConform: 0,
   }
 
-return elements.reduce(
+  return elements.reduce(
     (sum, { reviewProgress }) => ({
       totalReviewable: sum.totalReviewable + (reviewProgress?.totalReviewable || 0),
       doneConform: sum.doneConform + (reviewProgress?.doneConform || 0),
       doneNoneConform: sum.doneNoneConform + (reviewProgress?.doneNoneConform || 0),
     }),
     initial
   )
-  )
```

Comment:
Weird that you can remove one bracket and not break something. 🤷‍♂️

Reply:
I think that was broken in the branched this was branched from (and subsequently fixed, and just now merged to master), and after update to master i can't see that file in list of change)